lib/models/offering_wrapper.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import 'package_wrapper.dart';
+
+part 'offering_wrapper.freezed.dart';
+part 'offering_wrapper.g.dart';
+
+@freezed
+
+/// An offering is a collection of Packages (`Package`) available for the user
+/// to purchase. For more info see https://docs.revenuecat.com/docs/entitlements
+class Offering with _$Offering {
+  const Offering._(); // Added constructor
+  const factory Offering(
+    /// Unique identifier defined in RevenueCat dashboard.
+    final String identifier,
+
+    /// Offering description defined in RevenueCat dashboard.
+    final String serverDescription,
+
+    /// Array of `Package` objects available for purchase.
+    final List<Package> availablePackages, {
```

Comment:
perhaps it would be clearer for readers to have the `{` either as a separate line or as the first line in the first optional parameter, to clearly delineate the beginning of optional params? I had missed it at first, being at the end of this line

Reply:
Thoughts on this? @devasidmi

Reply:
It's vscode autoformatting, do u have any idea how to change that behaviour without changing editor settings?

Reply:
I don't know 🤷‍♂️ . But if that's how it's auto-formatted, let's leave it like that